src/main/java/com/example/book_your_seat/review/domain/Review.java

```diff
@@ -1,29 +1,29 @@
 package com.example.book_your_seat.review.domain;
 
+import com.example.book_your_seat.common.entity.BaseEntity;
 import com.example.book_your_seat.concert.domain.Concert;
 import com.example.book_your_seat.user.domain.User;
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.FetchType;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.persistence.JoinColumn;
-import jakarta.persistence.ManyToOne;
+import jakarta.persistence.*;
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Entity
 @Getter
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
-public class Review {
+@Table(indexes = {
+        @Index(name = "inx_reviews", columnList = "concert_id, created_at"),
+        @Index(name = "inx_user_create", columnList = "user_id, created_at")
+})
```

Comment:
오.. 신기하네요

Reply:
맞습니다 아까 설명이 부족해서 더 설명하자면 현재 쿼리에서
```suggestion
})List<Review> reviewList = queryFactory.selectFrom(review)
                .where(eqConcertId(concertId).and(ltReviewId(reviewId)))
                .join(review.user, user).fetchJoin()
                .limit(pageable.getPageSize() + 1) //후에 값이 더 있나 체크하기 위해 하나 더 가져옴
                .orderBy(review.createdAt.desc())
                .fetch();
```

where 문으로 @ManyToOne인 외래키를 조회하고 있는데 해당 외래키는 이미 인덱싱이 되어 있지만 카디널리티가 낮습니다. 그 이유는 여러개의 중복된 외래키가 존재해서 하나의 외래키를 조회할때 여러 컬럼을 중복조회 해야해서 성능이 안나오는거죠 그래서 OrderBy에 있는 createdAt은 중복이 낮은 컬럼이 이므로 함께 걸어줘서 카디널리티를 높여주는 방식을 사용했습니다. 그렇게 하면 카디널리티가 낮은 외래키를 높여줄 수 있습니다.